Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -186,13 +186,13 @@ namespace Multiplayer
                     mpComponent->GetNetworkEntityManager()->ClearAllEntities();
                 })
                 ->Method("GetCurrentBlendFactor", []()
+                {
+                    if (GetMultiplayer())
```

Comment:
You might as well save a pointer here, each GetMultiplayer goes through a mutex due to using AZ::Interface.

Reply:
Thank you! Fixed.